examples/inverse/plot_resolution_metrics_eegmeg.py

```diff
@@ -0,0 +1,135 @@
+"""
+==============================================================
+Compute spatial resolution metrics to compare MEG with EEGMEG.
```

Comment:
(nitpick) `EEGMEG` -> `EEG/MEG` or `EEG+MEG` or `M/EEG`.

Prior comment about periods in titles also applies here.